### ISSUE TITLE
WARN  [nuxt] Two component files resolving to the same name ProsePre

### ISSUE THREAD

**Issue by andrewtavis** [activist-org/activist#344]
### Terms

- [X] I have searched all [open bug reports](https://github.com/activist-org/activist/issues?q=is%3Aopen+is%3Aissue+label%3Abug)
- [X] I agree to follow activist's [Code of Conduct](https://github.com/activist-org/activist/blob/main/.github/CODE_OF_CONDUCT.md)

### Behavior

Currently the Docker build is not working for because of some unforeseen changes yarn.lock file. Is working for `yarn dev -o`, but `docker-compose up --build` is failing to make a connection.

Error for this:

```bash
nuxt_frontend   |  WARN  [nuxt] Two component files resolving to the same name ProseCode:
nuxt_frontend   | 
nuxt_frontend   |  - /app/node_modules/@nuxtjs/mdc/dist/runtime/components/prose/ProseCode.vue
nuxt_frontend   |  - /app/node_modules/@nuxt/content/dist/runtime/components/Prose/ProseCode.vue
```

I'm looking at this with @to-sta now. Making this issue for documentation :)

**Comment by andrewtavis** [comment]
@to-sta's addition of `Prettier` in #331 was problematic for who knows what reason 🤷‍♂️ Specifically for me I was getting the above issue and the data transfer to the frontend via ports had also changed. What I was seeing was:

<img width="641" alt="Screenshot_2023-09-05 15 12 16" src="https://github.com/activist-org/activist/assets/24387426/6564aa01-a700-4111-a11e-72dd9e4332bc">

What I'm seeing now that's fully functional is:

<img width="572" alt="Screenshot_2023-09-05 16 07 01" src="https://github.com/activist-org/activist/assets/24387426/bc3616d1-c7e2-40ba-a7ea-716d998e0e28">

Everything's working, so I'm going to close this as resolved once I send along the commit with the newly added dependencies.

CC @wkyoshida, maybe you have an idea on this, but also no stress!

**Comment by andrewtavis** [comment]
Closed via 65498d2 where the dependencies were added. Would be great to understand what was going on here, but we can figure it out later :) Closing now 😊
